WordPress/gutenberg
# feature: added aspect ration & scale option to video block

## What?
Added `Aspect ratio` & `Scale` controls to video block.

## Why?
Closes https://github.com/WordPress/gutenberg/issues/66933
Closes https://github.com/WordPress/gutenberg/issues/60911

## How?
- using existing `DimensionsTool` added aspect ratio & scale control to video block.

## Testing Instructions
1. open editor, insert video block and upload video
2. into inspector control check aspect ratio & scale controls to change video dimensions 
3. save post/page and same will reflect on front end


## Screenshots or screencast 

https://github.com/user-attachments/assets/095775de-9465-48da-86db-0374b21fef10
